templates/components/element_detail_tabs.html

```diff
@@ -0,0 +1,458 @@
+{% extends "base.html" %}
+{% load humanize %}
+{% load guardian_tags %}
+{% load static %}
+{% load q %}
+
+{% block title %}
+{{ portfolio.title }} Component Detail
+{% endblock %}
+
+{% block head %}
+{% include "controls/_style-controls.html" %}
+<style>
+  .project-image {
+    width: 20px;
+    float: left;
+    margin: 4px 0 5px 8px;
+  }
+
+  .inline {
+    display: inline;
+  }
+
+  h3 {
+    color: #888;
+    #font-weight: bold;
+    margin-top: 20px;
+  }
+
+  #above-tab-content, #tab-content {
+    max-width: 950px;
+    margin: auto;
+  }
+
+  .control-text {
+    white-space: pre-wrap;
+    font-size: 11pt;
+    /*max-width: 700px;*/
+    font-family: trebuchet ms, sans-serif;
+    text-align: justify;
+    /*text-justify: inter-word;*/
+    line-height: 24px;
+  }
+
+  #control-lookup input {
+    width: 130px;
+  }
+
+  #combined_smt {
+    width: 80%;
+    /*margin: 12px 0 16px 0px;*/
+    padding: 8px;
+    /*display: none;*/
+  }
+
+  textarea {
+    overflow-y: scroll;
+    resize: vertical;
+  }
+
+  a {
+    color: #666;
+  }
+
+  #common-tab-count, #component-tab-count {
+    display: inline-block;
+    color: #666;
+    background-color: #dedede;
+    border-radius: 20px;
+    line-height: 1;
+    padding: 2px 6px;
+    font-size: 0.8em; font-weight: 600;
+  }
+
+  .panel-heading-smt, .panel-heading-smt-body {
+    white-space: pre-line;
+  }
+
+  .panel-heading a:hover {
+    text-decoration: none;
+    color: #000;
+  }
+
+</style>
+{% endblock %}
+
+{% block contextbar %}{% endblock %}
+
+{% block body %}
+<div style="margin-top: 30px">
+  <div class="container">
+
+  <div class="row" style="">
+    <div id="above-tab-content" class="row">
+      <!-- Page Title-->
+      <div style="float: left;">
+          <h2 class="control-heading" style="">
+            <a href="/controls/components" style="color: black;">Component Library</a> &gt; <a href="/systems/{{system.id}}/components/selected" style="color: black;">Component Detail</a>
+          </h2>
+      </div>
+
+    </div>
+  </div><!--/row-->
+
+  <div id="above-tab-content" class="row">
+    <div style="float: left; padding: 0 0 0 0em;">
+        <h2 class="control-heading" style="">
+            {{ element.name }}
+        </h2>
+    </div>
+  </div>
+
+<div>&nbsp;</div>
```

Comment:
I'm curious if this needs its own div, or what this is here for.

Reply:
Same comment throughout. If this is being used as a spacing/styling thing, I wonder if we could remove these in favor of some styling?

Reply:
Agreed. There are definitely a bunch of these to address.

Reply:
@omarabed15 @kerryrm I recommend a ticket in Github for the styling items and use of spaces? We have some new resources coming on that could have this work. Maybe target January and lower priority?